spark-commons/src/test/scala/za/co/absa/spark/commons/errorhandling/implementations/submits/ErrorMessageSubmitWithoutColumnTest.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2023 ABSA Group Limited
+ * Copyright 2021 ABSA Group Limited
```

Comment:
on line 19, the `import org.apache.spark.sql.functions.{lit, map, typedLit}` -> `import org.apache.spark.sql.functions.lit` (the two others are not being used)